cake-autorate_setup.sh

```diff
@@ -10,7 +10,7 @@ set -eu
 # Set up remote locations and branch
 SRC_DIR="https://github.com/lynxthecat/cake-autorate/archive/refs/heads/"
 DOC_URL="https://github.com/lynxthecat/CAKE-autorate#installation-on-openwrt"
-BRANCH="testing"
```

Comment:
GitHub diff is incorrect. This file was not updated, just an article of me merging with your branch.